src/components/graph/graph.helper.js

```diff
@@ -23,7 +23,8 @@ import {
     forceX as d3ForceX,
     forceY as d3ForceY,
     forceSimulation as d3ForceSimulation,
-    forceManyBody as d3ForceManyBody
+    forceManyBody as d3ForceManyBody,
```

Comment:
Please remove this import. Is not used, and breaks lint as you can see on travis reports:

```
/home/travis/build/danielcaldas/react-d3-graph/src/components/graph/graph.helper.js
  27:21  error  'd3ForceCollide' is defined but never used  no-unused-vars
```